examples/tree_1d_dgsem/elixir_navierstokes_convergence_periodic_cfl.jl

```diff
@@ -0,0 +1,140 @@
+
+    p = rho * rho
+    p_t = 2.0 * rho * rho_t
+    p_x = 2.0 * rho * rho_x
+    p_xx = 2.0 * rho * rho_xx + 2.0 * rho_x * rho_x
+
+    E = p * inv_gamma_minus_one + 0.5 * rho * v1^2
+    E_t = p_t * inv_gamma_minus_one + 0.5 * rho_t * v1^2 + rho * v1 * v1_t
+    E_x = p_x * inv_gamma_minus_one + 0.5 * rho_x * v1^2 + rho * v1 * v1_x
+
+    # Some convenience constants
+    T_const = equations.gamma * inv_gamma_minus_one / Pr
+    inv_rho_cubed = 1.0 / (rho^3)
+
+    # compute the source terms
+    # density equation
+    du1 = rho_t + rho_x * v1 + rho * v1_x
+
+    # x-momentum equation
+    du2 = (rho_t * v1 + rho * v1_t
+           + p_x + rho_x * v1^2 + 2.0 * rho * v1 * v1_x -
+           # stress tensor from x-direction
+           v1_xx * mu_)
+
+    # total energy equation
+    du3 = (E_t + v1_x * (E + p) + v1 * (E_x + p_x) -
+           # stress tensor and temperature gradient terms from x-direction
+           v1_xx * v1 * mu_ -
+           v1_x * v1_x * mu_ -
+           T_const * inv_rho_cubed *
+           (p_xx * rho * rho -
+            2.0 * p_x * rho * rho_x +
+            2.0 * p * rho_x * rho_x -
+            p * rho * rho_xx) * mu_)
+
+    return SVector(du1, du2, du3)
+end
+
+volume_flux = flux_ranocha
+solver = DGSEM(polydeg = 3, surface_flux = flux_hllc,
+               volume_integral = VolumeIntegralFluxDifferencing(volume_flux))
+
+coordinates_min = -1.0
+coordinates_max = 1.0
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level = 4,
+                n_cells_max = 100_000)
+
+semi = SemidiscretizationHyperbolicParabolic(mesh, (equations, equations_parabolic),
+                                             initial_condition, solver,
+                                             source_terms = source_terms_navier_stokes_convergence_test)
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+tspan = (0.0, 10.0)
+ode = semidiscretize(semi, tspan)
+
+summary_callback = SummaryCallback()
+
+analysis_interval = 1000
+analysis_callback = AnalysisCallback(semi, interval = analysis_interval)
+
+alive_callback = AliveCallback(analysis_interval = analysis_interval)
+
+# Stepsize callback which selects the timestep according to the most restrictive CFL condition.
+# For coarser grids, linear stability is governed by the convective CFL condition,
+# while for high refinements (e.g. initial_refinement_level = 8) the flow becomes diffusion-dominated.
+stepsize_callback = StepsizeCallback(cfl = 2.7,
+                                     cfl_diffusive = 0.2)
+
+callbacks = CallbackSet(summary_callback,
+                        analysis_callback,
+                        alive_callback,
+                        stepsize_callback)
+
+###############################################################################
+# run the simulation
+
+sol = solve(ode, CKLLSRK95_4S(), adaptive = false, dt = 1.0,
```

Comment:
Maybe mention in a comment that `CKLLSRK95_4S` is a special integrator optimized for NS? I wondered why you didn't use `CarpenterKennedy2N54` until I dug into the docs for `OrdinaryDiffEqLowStorageRK`.